docs/source/remote_data.rst

```diff
@@ -199,6 +199,32 @@ Available expiry dates can be accessed from the ``expiry_dates`` property.
 
 .. _remote_data.enigma:
 
+Quandl
+======
+
+.. ipython:: python
+
+    import pandas_datareader.data as web
+    symbol = 'WIKI/AAPL'  # or 'AAPL.US'
+    df = web.DataReader(symbol, 'quandl', "2015-01-01", "2015-01-05")
+    df.ix['2015-01-04']
+
+.. _remote_data.quandl:
```

Comment:
put this right before the Quandl (and move the enigma one right before the Enigma)